src/legacy/core_plugins/expressions/public/np_ready/public/loader.ts

```diff
@@ -57,6 +63,12 @@ export class ExpressionLoader {
       this.render(data);
     });
 
+    this.params = {
+      searchContext: { type: 'kibana_context' },
+      extraHandlers: params.extraHandlers,
+    };
+
+    this.loadingSubject.next();
```

Comment:
I don't like that this is called in the constructor- see below for a suggestion on where to put it

Reply:
removing this for now.